### PR TITLE
feat: Upgrade to isort 5.12.0

### PR DESCRIPTION
**Pull request by AdarshRawat1**
I have used the isort Python utility/library to sort imports alphabetically and automatically separate them into sections and by type. We have also used the black profile to format the code.

### CODE REVIEW

**Comment by AdarshRawat1** [comment]
#2173

File: ci/common_sync/git-filter-repo
Comment (matejcik) [inline ci/common_sync/git-filter-repo:1]: please revert this

File: core/tests/test_apps.bitcoin.address.py [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 from apps.bitcoin.keychain import validate_path_against_script_type
 from apps.bitcoin.writers import *
+from apps.common import coins
+from trezor import wire
+from trezor.crypto import bip32, bip39
+from trezor.enums import InputScriptType
+from trezor.messages import GetAddress
+from trezor.utils import HashWriter
+
+from common import *
```
Comment (matejcik) [inline core/tests/test_apps.bitcoin.address.py:17]: we actually need "common" to be first. but maybe this is not worth worrying about, just reverting the whole of `core/tests` is good enough

File: core/tests/production_tests/main.py [hunk 2]
```diff
@@ -1,7 +1,6 @@
-import utime
-
 import trezorio as io
 import trezorui as ui
+import utime
```
Comment (matejcik) [inline core/tests/production_tests/main.py:3]: utime should also be in the "system builtins" group

File: tests/bip32.py [hunk 3]
```diff
@@ -24,7 +24,6 @@
 from ecdsa.curves import SECP256k1
 from ecdsa.ellipticcurve import INFINITY, Point
 from ecdsa.util import number_to_string, string_to_number
-
 from trezorlib import messages, tools
```
Comment (matejcik) [inline tests/bip32.py:27]: generally we want "trezorlib" to always be in the "1st party" section or whatever it is called, i.e., separated from click / pytest / ecdsa / etc.
there should be no diff here and in 90 % of `tests/`

File: setup.cfg [hunk 4]
```diff
@@ -19,17 +19,6 @@ ignore =
...
-[isort]
-multi_line_output = 3
-include_trailing_comma = True
-force_grid_wrap = 0
-combine_as_imports = True
-line_length = 88
-not_skip=__init__.py
-known_standard_library = micropython,ubinascii,ustruct,uctypes,utime,utimeq,trezorio,trezorui,trezorutils,trezorconfig,trezorcrypto
-known_third_party = curve25519,ecdsa,hypothesis,pytest,mnemonic,shamir-mnemonic,trezor,storage,protobuf
-known_first_party = trezorlib,apps
```
Comment (matejcik) [inline setup.cfg:31]: it looks like the new isort is good enough to figure out the "3rd party" stuff on its own, but we would like to keep both `known_standard_library` and `known_first_party`, see the other comments

**Review by matejcik — CHANGES_REQUESTED** [review]
thank you for your work, this is shaping up nicely.

please look into tweaking the isort config as indicated, and revert some of the changes. it might actually be better if you only include files covered by `make pystyle` (as opposed to every .py file in the project)

**Comment by AdarshRawat1** [comment]
@matejcik  I have made all the suggested changes.Please review!

**Comment by AdarshRawat1** [comment]
@matejcik 
I couldn't make necessary changes on master branch as there were too many files, so I created a separate PR from different branch.

**Pull request closed without merging**